libpod/adapter/runtime_remote.go

```diff
@@ -70,6 +75,30 @@ type remoteImage struct {
 	Runtime     *LocalRuntime
 }
 
+// Container ...
+type Container struct {
+	remoteContainer
+}
+
+// remoteContainer ....
+type remoteContainer struct {
+	ID         string
+	Image      string
+	ImageID    string
+	Command    []string
+	Created    time.Time
+	RunningFor string
+	Status     string
+	//Ports            []ocicni.PortMapping
+	RootFsSize int64
+	RWSize     int64
+	Names      string
+	Labels     []map[string]string
+	//	Mounts           []string
```

Comment:
Why are these commented out?

Reply:
they are going to be needed for ps when that is implemented.  I put in a TODO to clarify.